src/main/java/jenkins/scm/impl/subversion/SubversionSCMSource.java

```diff
@@ -159,6 +165,14 @@ public String getCredentialsId() {
     public void setCredentialsId(String credentialsId) {
         this.credentialsId = credentialsId;
     }
+    @DataBoundSetter
+    public void setAdditionalCredentials(List<AdditionalCredentials> additionalCredentials) {
+        this.additionalCredentials = additionalCredentials;
+    }
+    //without getter jelly will crash
+    public List<AdditionalCredentials> getAdditionalCredentials() {
+        return(additionalCredentials);
```

Comment:
Thank you for your comments. How do I make it restricted? Does adding "@Restricted(NoExternalUse.class)" work?